test/parallel/test-readline-interface.js

```diff
@@ -881,6 +882,56 @@ for (let i = 0; i < 12; i++) {
     rli.close();
   }
 
+  // Calling the promisified question
+  {
+    const [rli] = getInterface({ terminal });
+    const question = util.promisify(rli.question).bind(rli);
+    question('foo?')
+    .then(common.mustCall((answer) => {
+      assert.strictEqual(answer, 'bar');
+    }))
+    .catch(common.mustNotCall());
+    rli.write('bar\n');
+    rli.close();
+  }
+
+  // Aborting a question
+  {
+    const ac = new AbortController();
+    const signal = ac.signal;
+    const [rli] = getInterface({ terminal });
+    rli.on('line', common.mustCall((line) => {
+      assert.strictEqual(line, 'bar');
+    }));
+    rli.question('hello?', { signal }, common.mustNotCall());
+
+    ac.signal.addEventListener('abort', common.mustCall(), {
```

Comment:
Nit: this validation isn't needed here - since this tests AbortController and not rl.question (which has separate tests)

Reply:
Fixed in the last commit.